Gruntfile.js

```diff
@@ -140,6 +140,7 @@ module.exports = function( grunt ) {
 				"test/reporter-html/window-onerror-preexisting-handler.html",
 				"test/reporter-urlparams.html",
 				"test/moduleId.html",
+				"test/module-done.html",
```

Comment:
I'd prefer to avoid adding an additional test file, can you just update the existing tests, maybe [here](https://github.com/qunitjs/qunit/blob/63bcd15061fc706c9a058ad71c32ada8550ff39e/test/callbacks.js#L144)?